### PR TITLE
Add support for the report-uri directive

### PR DESCRIPTION
**Pull request by blazzy**
For reference:
  https://developers.google.com/web/fundamentals/security/csp/reporting?hl=en
  https://developer.mozilla.org/en-US/docs/Web/Security/CSP/Using_CSP_violation_reports

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,19 @@ csp := csp.New(csp.StarterConfig())
...
+
+Some browsers support sending CSP violation reports to a uri. You can specify
+this in the config with `ReportUri`.
+
+csp := csp.New(csp.Config{
+	Default:   csp.None,
+	Script:    csp.Self,
+	Connect:   csp.Self,
+	Img:       csp.Self,
+  ReportUri: "http://example.com/csp-violations",
```
Comment (yanfali) [inline README.md]: this doesn't look formatted quite right.

**Comment by yanfali** [comment]
Looks good, have you tested it on a live browser?

File: README.md (hunk 1 above)
Comment (blazzy) [reply README.md]: Argh. Tabs vs spaces. Looks good in my editor.

Comment (blazzy) [reply README.md]: Fixed.

**Comment by blazzy** [comment]
Yup, tested with our app.

```
ERRO[0009] sentry error                                  csp-report={"blocked-uri":"inline","colum...
```

**Comment by yanfali** [comment]
Thanks for finding the tabs/spaces issue. Please fix it in the PR and merge it after squash. Thanks

**Comment by blazzy** [comment]
Leaving in tabs as that is the `gofmt` standard.

**Pull request merged**